about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script>
+      src="https://kit.fontawesome.com/cb6c5078b1.js"
+      crossorigin="anonymous"
+      </script>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  <link rel="stylesheet" href="about.css">
+  <title>About Burning Circle Flight Academy</title>
+</head>
+<body>
+  <div class="top-section">
+    <div class="bar-container">
+    <ul class="bars">
+      <li>
+        <i class="fab fa-facebook-f"></i>
+      </li>
+      <li>
+        <i class="fab fa-twitter"></i>
+      </li>
+      <li>
+        <h4>English</h4>
+      </li>
+      <li>
+        <h4>My Page</h4>
+      </li>
+      <li>
+        <h4>Logout</h4>
+      </li>
+    </ul>
+    </div>
+  </div>
+  <header>
+    <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"><img src="images/plane3.svg" alt="plane" class="plane"></a>
+          <div class="menu-bar nav" id="menu">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Back to Home Page</a>
+          </div>
+        </nav>
+    </div>
+  </header>
```

Comment:
- [ ] The `header` element should be the same for the `Home` and about page, please update it.
Home page
![image](https://user-images.githubusercontent.com/33767951/141776604-07302bdd-6343-4931-9053-7138404d8747.png)
About page
![image](https://user-images.githubusercontent.com/33767951/141776974-9e9a8cee-4144-4c5b-bc5f-75861609a789.png)

Reply:
done